docs/asset-store/docs/03-02-bucket-cr-lifecycle.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Bucket Custom Resource Lifecycle
+type: Details
+---
+
+Read about the lifecycle of the Bucket Custom Resource (CR) and how other resources react to its creation and removal.
+
+
+## Create a Bucket CR
+
+When you create a Bucket CR, the Bucket Controller receives and event on the new CR and creates a bucket with the **name** specified in the CR is created in the Minio Gateway storage under the `ns-{NAMESPACE_NAME}-{CR_NAME}` location. The status of the CR contains a reference URL to the created bucket.
```

Comment:
```suggestion
When you create a Bucket CR, the Bucket Controller receives the event and creates a bucket with the **name** specified in the CR. It is created in the Minio Gateway storage under the `ns-{NAMESPACE_NAME}-{CR_NAME}` location. The status of the CR contains a reference URL to the created bucket.
```

`the event` - I'm not sure. But this is all about the "bucket CR has been created" event